front/src/routes/scene/edit-scene/actions/BlinkLightParams.jsx

```diff
@@ -0,0 +1,130 @@
+import { Component } from 'preact';
```

Comment:
It seems there is no default on the "blinking_speed", so when I create an action from scratch: 

![Screenshot 2024-02-26 at 09 16 33](https://github.com/GladysAssistant/Gladys/assets/7365207/c226987c-6ea1-4d48-ae5f-f099328e9f5f)

The select shows "lent" but it's not "lent", it's "empty", and it ends up in "200ms" (super fast)

Reply:
Fixed 
https://github.com/GladysAssistant/Gladys/pull/2007/files#diff-95d22c09953ef23db328da87b5b661548fe8958c05e5555874dd7ceed4a0841bR60-R62